com.avaloq.tools.ddk.test.core/src/com/avaloq/tools/ddk/test/core/StepResult.java

```diff
@@ -39,17 +39,18 @@ private StepResult() {
   }
 
   /** {@inheritDoc} */
+  @Override
   public T getValue() {
     return value;
   }
 
   /**
    * Sets the value of this {@link StepResult}. This can only be done by the test framework.
-   * 
+   *
    * @param value
    *          the new value for this {@link StepResult}
    */
-  protected void setValue(final T value) {
+  void setValue(final T value) {

```

Comment:
public?

Reply:
I think we explicitly don't want public here.